routes/web.php

```diff
@@ -33,4 +33,6 @@
     Route::delete('/profile', [ProfileController::class, 'destroy'])->name('profile.destroy');
 });
 
+Route::get('/chirps/past-week', [ChirpController::class, 'chirpsFromPastWeek'])->name('chirps.pastWeek');
```

Comment:
Add auth and verified middleware to this route to prevent non logged in users from being able to access this route by typing it directly in the url.

Reply:
I got you!